src/runboat/models.py

```diff
@@ -135,6 +135,10 @@ def slug(self) -> str:
     def deploy_link(self) -> str:
         return f"http://{self.slug}.{settings.build_domain}"
 
+    @property
+    def deploy_link_mailhog(self) -> str:
+        return f"http://{self.slug}.mail.{settings.build_domain}"
```

Comment:
Noting to myself that we have a leaking abstraction here. runboat does not know much about what is deployed by the kubefiles, except for these links to be displayed in the webui.